core/themes/basis/css/component/hero.css

```
@@ -26,6 +26,15 @@
   margin-top: -2rem;/* Negative top margin collapses spacing under header */
 }
 
+.update-1-30 .l-top .block-hero {
+  margin-top: 0;/* Negative top margin collapses spacing under header */
```

This code comment doesn't apply here.
```suggestion
  margin-top: 0;
```